addon/components/o-s-s/empty-state.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="fx-1 fx-col fx-xalign-center fx-gap-px-18">
+  {{#if (has-block "image")}}
+    <span class="">
```

Comment:
why the `span` here? as it's a named block, let's just let the caller handle the content here. same below for the actions